src/views/SystemIntake/ContractDetails/index.tsx

```diff
@@ -123,10 +123,7 @@ const ContractDetails = ({ systemIntake }: ContractDetailsProps) => {
       },
       costs: values.costs,
       contract: {
-        // ...values.contract,
-        hasContract: values.contract.hasContract,
-        contractor: values.contract.contractor,
-        number: values.contract.vehicle, // TODO - EASI-1977 - horrible hack to get this working with updated GQL schema
+        ...values.contract,
```

Comment:
Thanks for making sure to clean this up!

Reply:
Oh yea. Hard to pass a comment like that